src/Pages/ReviewWrite/ReviewWrite.js

```diff
@@ -1,7 +1,87 @@
-import React from 'react';
+import React, { useState } from 'react';
+import GrayMelon from './GrayMelon';
+import GreenMelon from './GreenMelon';
+import './ReviewWrite.scss';
 
 const ReviewWrite = () => {
-  return <div>reslissdfsdfsdfsdfst</div>;
+  const [input, setInput] = useState(10);
+  const [filed, setFiled] = useState([]);
+  // const reviewForm = document.querySelector('.ReviewWritePage');
+  // console.log(new FormData(reviewForm));
+  return (
+    <form className="ReviewWritePage">
+      <div className="ReviewWrite">
+        <div className="ReviewWriteSto">
+          <span className="ReviewWriteStoName">부촌육회 </span>
```

Comment:
```suggestion
          <span className="storeName">부촌육회 </span>
```
클래스 이름이 명확한 것은 좋지만 다소 길게 느껴집니다.
부모 태그에 reviewWrite라는 이름이 있기 때문에
해당 태그만 특정할 수 있는 storeName이 보다 간결하면서 의미를 잘 나타내겠네요!

이런 부분 고려해서 다른 클래스네임도 변경해주세요!